server/routes/admin.js

```diff
@@ -0,0 +1,9 @@
+const router = require("express").Router()
+const { generateCode } = require("../controllers/accessCode")
+const { createAdmin, verifyAdmin } = require("../controllers/admin")
+
+router.post("/accesscode", generateCode)
```

Comment:
Should this be a `get` route which is validated to make sure request is coming from admin?

Reply:
right it should be get. I will push the validation in the next pull request as I've added it in the new handlebars branch